src/param_curve.rs

```diff
@@ -92,7 +92,7 @@ pub trait ParamCurveArclen: ParamCurve {
         let mut t_last = 0.0;
         let mut t0 = 0.0;
         let mut t1 = 1.0;
-        let n = (-accuracy.log2()).ceil();
+        let n = (self.arclen(accuracy) / accuracy).log2().ceil();
```

Comment:
Yes, I think this is better logic. But to avoid a loop count of 0, I think it would be better to add a .max(1) on it.

Reply:
Yes, this code panics on large accuracy values. Pull request updated.